modules/apps/frontend-taglib/frontend-taglib/build.gradle

```diff
@@ -10,7 +10,6 @@ dependencies {
 	compileOnly project(":apps:frontend-taglib:frontend-taglib-clay")
 	compileOnly project(":apps:frontend-taglib:frontend-taglib-react")
 	compileOnly project(":apps:frontend-taglib:frontend-taglib-soy")
-	compileOnly project(":apps:portal-template:portal-template-react-renderer-api")
```

Comment:
No longer needed, I think...

Reply:
Yep! Since we aren't using React's ComponentTag more.